iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/converter/AbstractEncodingAwareAvroConverter.java

```diff
@@ -0,0 +1,39 @@
+package eu.dnetlib.iis.wf.importer.converter;
+
+import eu.dnetlib.iis.wf.importer.input.approver.ResultApprover;
+
+/**
+ * Encoding aware avro converter.
+ * @author mhorst
+ *
+ * @param <T>
+ */
+public abstract class AbstractEncodingAwareAvroConverter<T> extends AbstractAvroConverter<T> {
```

Comment:
Yes, the only purpose is to share field and getter method to entity builders. We could live without it - dropped.